### DISCUSSION TITLE
getting error "Invalid state message '{"batteryLevel":72,"batteryLife":72}"

### DISCUSSION THREAD

**Discussion by snigehere** [tsightler/ring-mqtt#635] [Support Q&A]
I am seeing errors in my log:

2023-05-17 21:20:55.482 WARNING (MainThread) [homeassistant.components.mqtt.sensor] Invalid state message '{"batteryLevel":72,"batteryLife":72}' from 'ring/-mask-/camera/-mask-/info/state'

Looking at the MQTT message in MQTT explorer I see 
 {
  "batteryLevel": 72,
  "batteryLife": 72,
  "firmwareStatus": "Up to Date",
  "lastUpdate": "2023-05-17T03:08:00Z",
  "wirelessNetwork": "nandf-home",
  "wirelessSignal": -45,
  "stream_Source": "rtsp://ring_mqtt_livestream:-mask--ring-mqtt:8554/-mask-_live",
  "still_Image_URL": "https://homeassistant:8123{{ states.camera.shed_snapshot.attributes.entity_picture }}"
}

Not sure what the issue is ... has the message format changed from Ring?
(edited as I had error from one camera and mqtt from other camera - corrected)

**Comment by tsightler** [comment]
Other than the warning message, what actual problem are you having?  The warnings seem likely caused by changes in HA, not anything to do with Ring.  Perhaps related to https://github.com/home-assistant/core/issues/89244.  I asked for logs from the user that opened https://github.com/tsightler/ring-mqtt/issues/634 but then they just closed the issue.

**Comment by snigehere** [comment]
I agree it seems to have been a transient issue and an issue on the ring side .. it started on the 15th and ended about 9pm UK time yesterday (17th). I had sensors for wifi signal and battery level go unavailable on my cameras ... this was picked up by an automation I run to alert me to any key sensors going unavailable .. when I first saw the issue I restarted the integration to see if that solved it but it did not - the first opportunity to look at the detail was later in the day yesterday ... and, of course, just as I started looking at logs it all resolved itself.  Thanks for your fast response.
